### ISSUE TITLE
Riot desktop app not opening the windows only showing a tab

### ISSUE THREAD

**Issue by jacobgonzales20** [vector-im/riot-web#13668]
Description

When i launch riot.im desktop app on windows 10 it pops up in a minimized format which i didn't leave it like that when i last had it open. It looks like this when i open riot https://imgur.com/a/LxSPK5n

Steps to reproduce

Kept riot open then restarted my pc then reopened riot

For the desktop app:

- Windows 
- Platform - desktop
- riot-web version: 1.6.0
- olm version: 3.1.3

**Comment by t3chguy** [comment]
Is this on a device with multiple screens? Windows has a tendency to sometimes put windows off-screen so Shift - Right Clicking that window in that task bar and clicking Maximise might bring it back into reality.

**Comment by jacobgonzales20** [comment]
No single screen

**Comment by IF-Adin** [comment]
Can confirm on multiscreen setup in Kubuntu: Riot starts minimized, opens normally. Session was saved with it not minimzed.

**Comment by t3chguy** [comment]
@IF-Adin are you using the `Start automatically after system login` option? If so that is intentional that it starts in the tray.

**Comment by IF-Adin** [comment]
@t3chguy Yes, i am. I wasn't aware that was the intended behavior.

**Comment by t3chguy** [comment]
Indeed this is intended behaviour, I originally wanted to add a separate checkbox for start minimized but there was pushback about adding too many options

**Comment by flatline-studios** [comment]
> Indeed this is intended behaviour, I originally wanted to add a separate checkbox for start minimized but there was pushback about adding too many options

I agree that too many options can be counter-intuitive... But, if an option makes sense, and is common in lots of other software, then it should 100% be in there.

“Everything should be made as simple as possible, but no simpler.” - That should apply to UI design and available options in general.
